### DISCUSSION TITLE
Cannot register - An unexpected error occurred. Please try again.

### DISCUSSION THREAD

**Discussion by jxs714** [HiEventsDev/Hi.Events#103] [Q&A]
Hey guys,

I've properly installed with Docker, setup the postgresql database_url and I cannot get passed the registration screen. It says "An unexpected error occurred. Please try again." every time I try to register. I do not see any errors in the logs. Am I supposed to have a working serial key in the docker environmental variables before I can begin using the software?

**Comment by daveearley** [comment]
Hi @jxs714,

No you don't require a serial key. That's odd you don't see any error in the logs. Have you looked at he Docker container logs? 

Can you please try he request again and take a screenshot of the response in the browser developer tools network tab.

Thanks

**Reply by jxs714** [reply]
Hi

Thanks for the update. The error in console says: POST https://DOMAIN.COM:8123/api/auth/register net::ERR_CONNECTION_REFUSED 
Also: Failed to load resource: net::ERR_CONNECTION_REFUSED  for /api/users/me

I am using NGINX with Plesk. Here is my nginx directive config:

location / {
	proxy_pass http://127.0.0.1:8123;
	proxy_set_header Upgrade $http_upgrade;
	proxy_set_header Connection "upgrade";
	proxy_set_header Host $host;
	proxy_set_header X-Real-IP $remote_addr;
	proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
	proxy_set_header X-Forwarded-Proto $scheme;
	proxy_set_header X-Forwarded-Host $host;
	proxy_set_header X-Forwarded-Port $server_port;
	proxy_buffering off;
	client_max_body_size 0;
	proxy_read_timeout 36000s;
	proxy_redirect off;
}

please advise

**Comment by jxs714** [comment]
Hi again,

I fixed the issue. It was because the URLs variables that were made by docker were incorrect. It needed to be the following:

VITE_FRONTEND_URL : https://domain.com
VITE_API_URL_CLIENT : https://domain.com/api
VITE_API_URL_SERVER : https://domain.com:8123/api
APP_CDN_URL : https://domain.com/storage

**Reply by FourHeadedDev** [reply]
bro i am currently getting the same issue i was fed up i tried hard for 5hrs can you please share the code from your .env file bcz in my it is not written VITE the things after VITE are written and in  my code app_cdn_url is also not written do we also have to write codes like MAIL_mailer mailport mail host 

please share what we have to write in .env 😭😭 i was literaaly about to cry then i found your comment please help am in 1st year
